package.json

```diff
@@ -21,7 +21,7 @@
     "test": "jest"
   },
   "devDependencies": {
-    "flow-bin": "^0.53.1",
-    "jest": "^20.0.4"
+    "flow-bin": "0.53.1",
+    "jest": "23.6.0"
```

Comment:
Can we use `^` dependency here instead of exact version just in case there are patches with bug fixes.

Reply:
👍